lstchain/calib/camera/tests/test_pedestals.py

```diff
@@ -0,0 +1,46 @@
+from ctapipe.utils import get_dataset_path
+
+import numpy as np
+from ctapipe.io import event_source
+from .. import PedestalIntegrator
+from .. import pedestals
+
+input_reader = event_source("/ctadata/franca/LST/LST-1.4.Run00167.0001.fits.fz", max_events=10)
```

Comment:
This needs to be changed to use an accessible file (prob. in `cta-lstchain-extra`)

Reply:
Yes, the problem is that I miss a data file small enough to be put into the extra directory, I asked already for it, let ask again.

Reply:
Ok, thank you.
In this state, the tests would never pass.

Reply:
Should we suspend it for the moment?

Reply:
If the tests are passing on your computer and this module is important and we don't want anymore, maybe we can comment the tests. In this case open an issue to reactivate tests later and solve it asap.